client/components/TestQueue/index.jsx

```diff
@@ -253,6 +253,7 @@ const TestQueue = () => {
 
             {isAdmin && (
                 <ManageTestQueue
+                    enableManageRequiredReports={true}
```

Comment:
I don't follow why this prop is necessary. Is there a case when a viewer is admin and looking at `ManageTestQueue` but should not be seeing the "Manage Required Reports" section?

Reply:
Hey @stalgiag that prop makes it so that the `Manage Required Reports` dropdown only shows up on the `Data Management` page and not also on the `Test Queue` page.